controller/src/beerocks/master/son_management.cpp

```diff
@@ -30,6 +30,34 @@ using namespace beerocks;
 using namespace net;
 using namespace son;
 
+//////////////////////////////////////////////////////////////////////////////
+/////////////////////////// Local Module Functions ///////////////////////////
+//////////////////////////////////////////////////////////////////////////////
+
+/**
+ * @brief Set the channel pool to contain all supported channels.
+ * @param channel_pool_set:  Set containing the current channel pool.
+ * @param channel_pool_size: Size of the given set.
+ * @param database:          Reference to the DB.
+ * @param radio_mac:         MAC address of radio.
+ */
+static void set_pool_to_all_supported_channels(std::unordered_set<uint8_t> &channel_pool_set,
```

Comment:
instead of `set_pool_to_all_supported_channels` I would suggest a `get_pool_of_all_supported_channels` which is what you do.

Reply:
Agree, much better name,
Changed